content/docs/languages/rust/40-supply-chain-analysis.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Supply chain analysis"
+slug: rust-supply-chain-analysis
+summary: "This section describes tricks for Rust unit testing"
+weight: 40
+---
+
+# Supply chain analysis
+
+## Vetting
+
+The tools in this section are more for "understanding" than "checking." E.g., running them does not produce "bug reports", but can help you assess maturity and security of dependencies. Tools below are rather "quantitative" than "qualitative" - you will need to do manuall, in-depth review of the outputs to extract any solid evidences about the maturity.
+
+ Run cargo-supply-chain
+This reveals who you are implicitly trusting when you rely on a dependency (e.g., you want that set to be small)
+ Run cargo-vet
+This checks if dependencies were audited by a "trusted party"
+rust-crate-audits - collection of Google's audits
+ Run cargo-crev
+This is a distrubuted core-review platform
+ Run cargo-deny
+Cargo plugin for linting your dependencies
+
+
+
+## Looking for vulnerabilities
+
+The ultimate tool for detection of vulnerabilities is `cargo-audit` - you should just use it.
+The [`cargo-audit` compares dependencies]() against a database with known vulnerabilities:
```

Comment:
I would link `cargo-audit` on the previous line, and write "The tool compares..." on this line.